is/identity-server-deployment.yaml

```diff
@@ -67,6 +73,9 @@ spec:
           mountPath: /home/wso2carbon/wso2is-5.5.0/repository/deployment
         - name: shared-tenants-persistent-disk
           mountPath: /home/wso2carbon/wso2is-5.5.0/repository/tenants
+        # unocomment following to mount the repository directory to NFS
+        # - name: is-additional-config-storage-volume
```

Comment:
let's change "is-additional-config-storage-volume" to "is-additional-changes-storage-volume", WDYT? since it's not just configs....